libkbfs/conflict_resolver.go

```diff
@@ -3770,7 +3770,7 @@ outer:
 		return err
 	}
 
-	handle := cr.fbo.getHead(lState).GetTlfHandle()
+	handle := cr.fbo.getTrustedHead(lState).GetTlfHandle()
```

Comment:
seems like we should be checking here for a nil head, or somehow plumbing the name/public info through from the caller.